addons/mail/static/src/components/message_reaction_per_emoji/message_reaction_per_emoji.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MessageReactionPerEmoji" owl="1">
+        <div t-on-click="_onClick" class="o_MessageReactionPerEmoji" t-att-class="{'o-active': hasUserReacted}" t-att-title="reactionsSummary">
+            <span class="o_MessageReactionPerEmoji_emojiUnicode">
+                <t t-esc="emojiUnicode"/>
+            </span>
+            <span class="o_MessageReactionPerEmoji_count" t-att-class="{'o-activeCount': hasUserReacted}">
```

Comment:
Prefer classname as close to model terminology as possible.
Here `hasUserReacted` => `o-hasUserReacted`.